lms/static/sass/course/_profile.scss

```diff
@@ -169,6 +169,111 @@
         }
       }
     }
+    div.upgrade-banner {
+      // This banner uses the Pattern Library's defined variables
+      @include border-left(0px);
+
+      border: 1px solid $lms-border-color;
+      width: 100%;
+      display: table;
+      margin-bottom: 20px;
+
+      .notification-color-border {
+        width: 6px; //Value defined by UX team
+        min-height: 100%;
+        margin: 0;
+        display: table-cell;
+        background: $notification-highlight-border-color;
+      }
+
+      .notification-content {
+        display: inline-flex;
+        align-items: center;
+        align-content: flex-start;
+        flex-flow: row wrap;
+        background: $notification-background;
+        width: 100%;
+        padding: $baseline/2 0;
+        margin-bottom: 0;
+        justify-content: space-between;
+
+        .upgrade-icon {
+          margin: 0;
+          padding: $baseline/2 $baseline;
+          flex-flow: row nowrap;
+          align-items: center;
+          // flex: grow, shrink, base
+          // The 7 was the value that allowed the icon image to grow to the UX
+          // desired size.
+          flex: 7 1 50px;
+          // The following dimensions were added so that the
+          // icon will adjust as the notification is adjusted
+          // but will not be smaller or larger than UX requirements.
+          min-height: 50px;
+          min-width: 80px;
+          max-height: 90px;
+          max-width: 130px;
+
+          img {
+            min-height: 50px;
+            min-width: 80px;
+          }
+        }
+
+        .upgrade-msg {
+          flex: 5 1 60%; //This percentage was required to get the text
```

Comment:
I strongly dislike fudging percentages until the UX looks right in one browser at one zoom level. Flexbox is very good at letting you define fixed areas and dynamic areas, so you shouldn't need to do this. Can we chat next week about the options.